### ISSUE TITLE
Power-level text in `MemberList` is not very accessible

### ISSUE THREAD

**Issue by ara4n** [vector-im/element-web#12557]
New power-level (PL) text in `MemberList` is not very accessible. It's very small font size, and very low contrast.

How about just putting the PL info physically where the presence info goes (when non-default PL is set)?

**Comment by MadLittleMods** [comment]
Member list for reference as of 2022-08-25,

![](https://user-images.githubusercontent.com/558581/186758475-677f231d-b60b-485b-ac84-0d4e6f7d6e9c.png)

Which fails the AA and AAA contrast ratios at that font-size,

![](https://user-images.githubusercontent.com/558581/186758638-6dcda518-a4ea-4a55-89ed-00fd7e204c5a.png)

**Comment by germain-gg** [comment]
Fixed by https://github.com/matrix-org/matrix-react-sdk/pull/11079

<img width="753" alt="Screenshot 2023-07-19 at 09 08 42" src="https://github.com/vector-im/element-web/assets/769871/28526b5b-492e-4cfa-bfcd-6b7b3744c480">
